src/test/groovy/func/GradleCompatibilitySpec.groovy

```diff
@@ -30,14 +30,26 @@ class GradleCompatibilitySpec extends GatlingFuncSpec {
     }
 
     @Unroll
-    void 'should fail for version #gradleVersion that is less than 5.0'() {
+    void 'should fail with friendly message for version #gradleVersion that is less than 5.0'() {
         given:
         prepareTest()
         when:
         BuildResult result = executeGradleTaskWithVersion('tasks', gradleVersion, true)
         then:
         result.output.contains("Current Gradle version (${gradleVersion}) is unsupported. Minimal supported version is 5.0")
         where:
-        gradleVersion << ["4.0.1", "4.10.2", "3.5", "3.0", "2.9"]
+        gradleVersion << ["4.10.2"]
+    }
+
+    @Unroll
+    void 'should fail with exception for version #gradleVersion that is less than 4.10'() {
```

Comment:
@slandelle I think this is acceptable, WDYT?

Reply:
yes